tcms/xmlrpc/api/testcaserunstatus.py

```diff
@@ -4,7 +4,7 @@
 
 from modernrpc.core import rpc_method
 
-from tcms.testruns.models import TestCaseRunStatus
+from tcms.testruns.models import TestExecutionStatus
```

Comment:
Same note about file rename and backwards/forwards compatible method decorators.